src/test/java/com/hazelcast/kubernetes/KubernetesClientTest.java

```diff
@@ -56,12 +56,16 @@
 
     @Before
     public void setUp() {
-        String kubernetesMasterUrl = String.format("http://%s:%d", KUBERNETES_MASTER_IP, wireMockRule.port());
-        kubernetesClient = new KubernetesClient(NAMESPACE, kubernetesMasterUrl, TOKEN, CA_CERTIFICATE, RETRIES);
+        kubernetesClient = newKubernetesClient(false);
         stubFor(get(urlMatching("/api/.*")).atPriority(5)
                                            .willReturn(aResponse().withStatus(401).withBody("\"reason\":\"Forbidden\"")));
     }
 
+    private KubernetesClient newKubernetesClient(boolean useNodeNameAsExternalAddress) {
```

Comment:
1. Make it static
2. Move this method down, since it's used in multiple places in this class.

Reply:
The method `newKubernetesClient` uses `wireMockRule`, which I believe cannot be made static, so unfortunately I cannot make the method static unless we provide `wireMockRule` as a parameter.

Which version do you prefer: static method with `wireMockRule` as parameter; or non-static method?

Reply:
It's only the port number we need from `wireMockRule`, so I can provide the port number as a parameter and make the method static. Then the caller needs to know how to find the port number...

Reply:
Left it non-static, moved it down to the other helper methods.